### ISSUE TITLE
cd loss explode

### ISSUE THREAD

**Issue by xljh0520** [diviswen/Cycle4Completion#6]
Hi，thanks for sharing your code!
I ran your code under your default hyperparameter with command
```
python main_code.py
```
The cd loss in log explode like this.
![1630980715(1)](https://user-images.githubusercontent.com/56757835/132273675-090961c1-294b-499d-a7a3-bf4870e7e769.png)
Could you please tell me the reason?

**Comment by diviswen** [comment]
We also observed the same problem, and the directly reason is that the gradients suddenly rise in certain epoch.
Currently we are still focusing on it to stabilize the training. There are two classes (table, sofa) easy to collapse in early stage.

By the way, we actually didn't train the model to the max_epoch=500  as stated in "main_code.py". Usually 200~300 epoch would suffice the training.
